bin/alohomora

```diff
@@ -118,24 +133,24 @@ class Main(object):
 
         if not DURATION_MIN <= duration <= DURATION_MAX:
             alohomora.die("Duration of '%s' not in the range of %s-%s seconds" %
-                    (self._get_config('duration', None),
-                     DURATION_MIN,
-                     DURATION_MAX))
+                          (self._get_config('duration', None),
+                           DURATION_MIN,
+                           DURATION_MAX))
 
         #
         # Get the user's credentials
         #
         username = self._get_config('username', os.environ["USER"])
-        if(not username):
+        if not username:
             alohomora.die("Oops, don't forget to provide a username")
 
         password = getpass.getpass()
 
         idp_url = self._get_config('idp-url', None)
-        if(not idp_url):
+        if not idp_url:
             alohomora.die("Oops, don't forget to provide an idp-url")
 
-        auth_method = self._get_config('auth-method', None)
+        auth_method = self._get_config('auth_method', None)
```

Comment:
The readme example shows the 'auth-method' as param in the config file, is there any place parse the config to be 'auth_method'?

Reply:
Both argparse and Click, our CLI library, automatically change `-` characters to `_` when doing variable access.  So while the user types the `-` version, in code we can only ever refer to them by the `_` version.

Reply:
EDIT: turns out I was wrong!  I need to do some more reading, it turns out.  Good catch @zhanjing1214!

Reply:
The pattern I chose to follow was "all dashes."  This means that you type the configuration variable the same way, whether you're using the CLI or the config file.  The internal conversion that Click does for CLI arguments is handled internally, to avoid confusing users.